app/main.py

```diff
@@ -1 +1,66 @@
-# write your code here
+class SoftwareEngineer:
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.extend([skill])
+
+
+class FrontendDeveloper(SoftwareEngineer):
+
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.name = name
+        self.skills.extend([
+            "JavaScript",
+            "HTML",
+            "CSS",
+        ])
```

Comment:
It's not required, but you can write this in 1 line, it won't be really long here (not required to fix, if you disagree)

Reply:
I'll leave it like it was, cause if there is a longer lists - code would look different. It's could make it not easy to read at some point.